interface-definitions/include/openfabric/interface.xml.i

```diff
@@ -0,0 +1,86 @@
+<!-- include start from openfabric/interface.xml.i -->
+<tagNode name="interface">
+  <properties>
+    <help>Interface params</help>
+    <completionHelp>
+      <script>${vyos_completion_dir}/list_interfaces</script>
+    </completionHelp>
+  </properties>
+  <children>
+    <leafNode name="csnp-interval">
+      <properties>
+        <help>Set CSNP interval</help>
```

Comment:
What's CSNP? Can you add a bit more clarity to the help string by expanding the abreviation?